dataactbroker/app.py

```diff
@@ -21,6 +21,13 @@
 from dataactcore.utils.responseException import ResponseException
 from dataactcore.utils.statusCode import StatusCode
 
+# Datadog import
+USE_DATADOG = False
```

Comment:
Add a comment here to let developers know that this value is being changed with Ansible, so nobody deletes it thinking the next `if` can never be reached.